src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/DryRunNotificationService.kt

```diff
@@ -140,7 +113,7 @@ class DryRunNotificationService(
     item: CsrModifiedDetailDto,
     first: Boolean
   ): CsrModifiedDetailDto =
-    if (first) item else if (item.shiftModified!!.isAfter(acc!!.shiftModified)) item else acc
+    if (first) item else if (item.shiftModified?.isAfter(acc!!.shiftModified) == true) item else acc!!
```

Comment:
not sure why i now need  acc!!

Reply:
You could probably make this cleaner by the  starting "if" being:

`if (first && acc != null && its.shiftModfied != null)` since that is exactly what you are expecting, then you won't need all the ?!! etc

Reply:
But you should then question why the `acc` parameter is nullable when you are assuming it isn't with the `acc!!`